framework/infrastructure/gcp/compute.py

```diff
@@ -61,10 +62,12 @@ def __init__(
         api_manager: gcp.api.GcpApiManager,
         project: str,
         gfe_debug_header: Optional[str] = None,
+        add_dualstack_support: bool = False,
         version: str = "v1",
     ):
         super().__init__(api_manager.compute(version), project)
         self.gfe_debug_header = gfe_debug_header
+        self.add_dualstack_support = add_dualstack_support
```

Comment:
This is only used in `create_backend_service_traffic_director`. Let's make it a parameter to this method instead. No need for the object property.

Reply:
Done